4-projects/business_unit_1/development/example_peering_project.tf

```diff
@@ -0,0 +1,72 @@
+/**
+ * Copyright 2020 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+
+locals {
+  vpc_name      = "${local.env_code}-peering-base"
+  network_name  = "vpc-${local.vpc_name}"
+  env_code      = "d"
+  environment   = "development"
```

Comment:
Can we follow the convention we are using elsewhere, where these are hard coded. e.g

https://github.com/terraform-google-modules/terraform-example-foundation/blob/master/4-projects/business_unit_1/development/example_base_shared_vpc_project.tf

These values are not reused, so it does not provide much value and can make the code harder to read